SU-SWS/stanford_access_control
# SNI-71: Refactor grants of access control module.

# READY FOR REVIEW

# Summary
- Refactored the grants to be specifically role based. 

# Needed By (Date)
- asap

# Urgency
- 7/10

# Steps to Test

1. Clone jumpstart-neuro somewhere you can make changes to it
2. Check out this branch
3. Edit a node that also appears in the news view
4. Add a SAC rule that only allows admins to view that node by editing the node
5. Validate that the node doesn't show up in the view nor can you directly navigate to it with an anonymous user
6. Validate that the same node is available to admins
7. Add a blanket wildcard rule for news items `news/*` for only admin access
8. Rebuild permissions
9. Validate that only admins see news items.

# Affected Projects or Products
- Neuro

# Associated Issues and/or People
- SNI-71

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
